compiler/optimizer/OMRValuePropagation.cpp

```diff
@@ -3871,257 +3871,12 @@ int32_t TR::GlobalValuePropagation::perform()
 
 void OMR::ValuePropagation::getParmValues()
```

Comment:
I find it odd that a function which looks like a getter (`get*`) has a `void` return value.

Reply:
I don't disagree.  I'll defer to those with more intimate knowledge of this code to suggest a better name, but it will require a dance with OpenJ9 to rename that I'll be happy to do in a separate PR.

Reply:
I'm fine with opening up an issue to address this later, perhaps by beginner contributors?

Reply:
get is the right name for this right now - the results are sorted in `_parmValues` which is an implicit convention. The VP code is littered with stuff like this so I don't want to change it in one place without a view of how to handle it more broadly.